extensions/widgets/segmented/segmented.lcb

```diff
@@ -287,7 +290,6 @@ public handler OnCreate()
 	put (the number of elements in mSegmentNames) into mNumSegments
 	put the empty path into mPerimeter
 	put the empty list into mLines
-	put 0 into mRadius
```

Comment:
@jameshale the documented default is 15 so I think you should set that here

Reply:
ok, but does this mean that...
1326     metadata cornerRadius.default is "15"
is superfluous?

Reply:
Not quite, no - if you create the widget in script, nothing happens with the metadata. The metadata default is used by the IDE as the property default (so for example if we added a 'restore all defaults' button to the property inspector, we'd need to use the provided metadata value).

At some point it might be possible to define a constant which is used in both the OnCreate handler and a metadata declaration, but not at the moment unfortunately.

Reply:
@livecodeali ah thanks. I thought more about this later that night and wondered if something like this was the case.